docs/articles/documentation/extending-testcafe/index.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: docs
+title: Extending TestCafe
+permalink: /documentation/extending-testcafe/
+---
+# Extending TestCafe
+
+You can extend the TestCafe functionality with custom plugins.
```

Comment:
Are you sure we need to say about custom plugins right here - in the very introductory sentence?
It's just one of the means to achieve our goals and we can keep it for more detailed topics.
What do you think?